src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,3 +1,20 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => <>Put the comment here</>;
+export const CommentInfo: React.FC<{ comment: Comment }> = ({ comment }) => {
```

Comment:
It's always better to extract properties from a prop here on at the line below:

`{ comment: { name,  email ...} }`